logzio/handler.py

```diff
@@ -102,14 +102,16 @@ def add_to_buffer(self, message):
             self.logs_counter_condition.release()
 
     def handle_exceptions(self, message):
-        if message.exc_info:
-            return '\n'.join(traceback.format_exception(*message.exc_info))
-        else:
-            return message.getMessage()
+        message = '\n'.join(traceback.format_exception(*message.exc_info))
```

Comment:
there's an obvious problem here.. this method doesn't return the generated message..